test/fluree/db/property_class_statistics_test.clj

```diff
@@ -246,14 +247,42 @@
 
             info     @(fluree/ledger-info conn ledger-id)]
 
-        (testing "ledger-info includes standard fields"
-          (is (some? (:address info)) "Should have address")
-          (is (some? (:alias info)) "Should have alias")
-          (is (some? (:branch info)) "Should have branch")
-          (is (some? (:t info)) "Should have t")
+        ;; Print full ledger-info for inspection
+        (println "\n=== FULL LEDGER-INFO STRUCTURE ===")
```

Comment:
I think the whole ledger-info response is deterministic outside of `:commit` `:time`. We can use a `with-redef` to hardcode the time produced by `util/current-time-millis` and get a fully deterministic response and then we can just expect the response, instead of having all these individual assertions about the shape.

Reply:
Addressed in 6b81a321aeb4fb50cf3058bfd408635ea3fc5744